src/brevitas/export/onnx/qonnx/handler.py

```diff
@@ -14,14 +14,95 @@
 from brevitas.proxy import DecoupledWeightQuantProxyFromInjector
 from brevitas.proxy import DecoupledWeightQuantWithInputProxyFromInjector
 from brevitas.proxy import WeightQuantProxyFromInjector
+from brevitas.proxy.float_parameter_quant import WeightFloatQuantProxyFromInjector
+from brevitas.proxy.float_runtime_quant import ActFloatQuantProxyFromInjector
 from brevitas.proxy.runtime_quant import TruncQuantProxyFromInjector
 
 from .function import BrevitasBinaryQuantFn
+from .function import BrevitasFloatQuantFn
 from .function import BrevitasQuantFn
 from .function import BrevitasQuantLSTMCellFn
 from .function import BrevitasTruncFn
 
 
+class BrevitasFloatQuantProxyHandler(ONNXBaseHandler, ABC):
+
+    def validate(self, module):
+        assert not module.is_groupwise, "Export with Per Group quantization not supported"
```

Comment:
is this a fundamental limitation or just not implemented yet? the `FloatQuant` custom op does support broadcastable shapes for the scale, bitwidth.. inputs

Reply:
I think it's a residual of when groupwise didn't have its own proxy, and it requires some special care.

In principle there are limitations for groupwise export but not for the QONNX. However, we would need a separate class to handle groupwise, mostly because shaping of tensors/scales is special compared to per_tensor/per_channel case.

I would do it in a separate PR, but let me know if it's "urgent"

Reply:
Got it, thanks! I don't think this is urgent yet.